### PR TITLE
Cypress Tests for AI SiteGen Preview Step

### PR DESCRIPTION
**Pull request by avneet-raj**

### CODE REVIEW

File: tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    it( 'Check for the favourited theme versions', () => {
+        cy.get('g[clip-path="url(#heart-filled_svg__a)"]').should('not.exist');  // when no fav theme is selected
+        cy.get( '.live-preview-sitegen--selectable-card__live-preview-container-buttons__button__icon' )
+            .eq(0)
+            .scrollIntoView()
+            .should('be.visible')
+            .click();
+        cy.get('g[clip-path="url(#heart-filled_svg__a)"]' , {timeout:10000} )
+            .should('exist');
+    } );
```
Comment (sangeetha-nayak) [inline tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js:50]: Are wen not verifying if favorited theme in preview page is also favorited in editor page?

File: tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    it( 'Select any theme and go forward to the next step', () => {
+        cy.get('.live-preview-sitegen--selectable-card')
+            .eq(0)
+            .scrollIntoView()
+            .click();
+        cy.wait(5000);
+        cy.url().should('not.contain', 'sitegen/step/preview');
+    } );
+});
```
Comment (sangeetha-nayak) [inline tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js:79]: Should we add a test for favorite icon animation at the bottom of preview page?

Comment (avneet-raj) [reply tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js:79]: cy.get('g[id="State\\=Active"]').should('exist');

this I've already added.

File: tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js (hunk 1 above)
Comment (avneet-raj) [reply tests/cypress/integration/5-AI-SiteGen-onboarding-flow/preview.cy.js:60]: updated

**Review by sangeetha-nayak — APPROVED** [review]
LGTM

**Pull request merged**